openlibrary/macros/LoanStatus.html

```diff
@@ -113,7 +113,9 @@
         <a href="https://openlibrary.org/bookdrive" target="_blank">$_('Learn More')</a>
       </p>
   $else:
-    <a href="$work_key" class="cta-btn cta-btn--missing" data-ol-link-track="CTAClick|NotInLibrary">$_('Not in Library')</a>
+    <div class="cta-button-group">
+      <a href="$work_key" class="cta-btn cta-btn--missing" data-ol-link-track="CTAClick|NotInLibrary">$_('Not in Library')</a>
+    </div>  
```

Comment:
Please remove the trailing whitespace...
```suggestion
    </div> 
```

Reply:
This appears to be working properly on my local instance:

![Screenshot from 2021-12-08 12-14-57](https://user-images.githubusercontent.com/28732543/145253381-dceddade-3d8e-4f2f-807a-a8cb634cac84.png)

I'll be happy to merge this once the trailing whitespace is removed.